sliceflag.go

```diff
@@ -229,7 +229,7 @@ func unwrapFlagValue(v flag.Value) flag.Value {
 
 // NOTE: the methods below are in this file to make use of the build constraint
 
-func (f *Float64SliceFlag) SetValue(slice []float64) {
+/*func (f *Float64SliceFlag) SetValue(slice []float64) {
```

Comment:
Did you want to address these commented-out slice type bits within this PR or leave it as later work?

Reply:
I will remove these commented out slice types